File: NAI/Android/src/dk/itu/nai/authentication/ClientCredentials.java
package dk.itu.nai.authentication;

public abstract class ClientCredentials {
	private String userId;

	abstract public byte[] toByteArray();
	
	public ClientCredentials(String userId) {
		this.userId = userId.trim();
	}
	
	public String getUserId()
	{
		return userId;
	}
}

File: trunk/NAI/Surface/NAI/Client/Authentication/ClientCredentials.cs

namespace NAI.Client.Authentication
{
    public abstract class ClientCredentials
    {
        public string UserId { get; private set; }

        public ClientCredentials(string userId)
        {
            this.UserId = userId;
        }

        public override string ToString()
        {
            return UserId;
        }
    }
}
